tests/testthat/test-acls.R

```diff
@@ -2,7 +2,7 @@
 test_that("renv_acls_reset() can reset ACLs", {
 
   skip_on_cran()
-  skip_if(!renv_platform_linux())
+  skip_on_os(c("windows", "mac"))
```

Comment:
This test would fail on non-Linux Unix machines so it really is intended to be "Linux-only" rather than not-Windows / not-macOS.

Reply:
In other words, `skip_unless_os("linux")`?

Reply:
I also don't like `skip_on_os()` because I can never remember if macOS is "mac", "macos", "Darwin", or whatever else. I know I can look at the documentation but autocompletion fills in the blank if I have `skip_on_macos()` or something.

Reply:
Can you explain more the distinction between unix and linux here? I didn't think we ran tests on non-Linux unix platforms so I don't understand why the distinction is important.

FWIW you can just call `skip_on_os("foo")` and the error message will help you 😄